packages/venia-concept/src/components/Checkout/form.css

```diff
@@ -18,6 +18,19 @@
     text-transform: capitalize;
 }
 
+.paymentDisplayPrimary {
+    text-transform: capitalize;
+}
+
+.paymentDisplaySecondary {
+    /* The ::first-letter pseudo element below only works on block elements */
```

Comment:
Interesting approach :P My mind immediately jumps to i18n and rtl languages but not sure if we care for Venia or even if that matters. Is there capitalization in rtl? Would browsers apply `::first-letter` in rtl to the right-most character?

![](https://weeklydevotion.files.wordpress.com/2014/06/whoa.jpg?w=300&h=300)

Reply:
I found this [CSS Tricks](https://css-tricks.com/almanac/selectors/f/first-letter/) post informative